Lifecycle customization (#592)
DAGWorks-Inc/hamilton: docs/reference/customizing-execution/index.rst

==============
GraphAdapters
==============

This section helps determine ways to execute Hamilton.
Note that these are special cases of the Lifecycle Customizations described in the next section,

> meant to help with execution. They implement multiple lifecycle customizations at once.

Reviewer: ```suggestion
meant to help with execution. They implement multiple lifecycle customizations in a single place.
```
